man/asciidoctor.1

```diff
@@ -178,23 +178,27 @@ If this option is not set (default: FATAL), the program exits with a status code
 .sp
 \fB\-q, \-\-quiet\fP
 .RS 4
-Silence warnings.
+Silence all messages.
```

Comment:
all application messages ? or all messages ? even if I enable the pedantic warnings ?

Reply:
Perhaps we should just say "no output".

Though, there's another open issue that proposed to break this down even further. See https://github.com/asciidoctor/asciidoctor/issues/2876.

Reply:
Indeed it's a `silent` mode. Until we split the `--quiet` option, we could add a "no output":

> Silence all messages. (no output)

Reply:
:+1: 

I'm going to go with "Don't output messages" since technically you can still output the document to STDOUT.